README.md

```diff
@@ -62,6 +57,48 @@ Objectives:
 - Extensible: Customizable without touching the core code.
 - Unified: Single codebase, deployable as an agent or collector with support for traces, metrics and logs.
 
+## Status
+
+The collector components and implementation are in different stages of stability, and usually split between
+functionality and configuration:
+
+| Component | Status |
+|---|---|
+| OTLP traces protocol | Stable |
+| OTLP metrics protocol | Stable |
+| OTLP logs protocol | Beta |
+| OTLP traces receiver functionality | Stable |
+| OTLP metrics receiver functionality | Stable |
+| OTLP logs receiver functionality | Beta |
+| OTLP traces receiver configuration | Beta |
+| OTLP metrics receiver configuration | Beta |
+| OTLP logs receiver configuration | Beta |
+| OTLP traces exporter functionality | Stable |
+| OTLP metrics exporter functionality | Stable |
+| OTLP logs exporter functionality | Beta |
+| OTLP traces exporter configuration | Beta |
+| OTLP metrics exporter configuration | Beta |
+| OTLP logs exporter configuration | Beta |
+| Logging exporter | Experimental |
+| Batch processor functionality | Beta |
+| Batch processor configuration | Beta |
+| MemoryLimiter processor functionality | Beta |
+| MemoryLimiter processor configuration | Beta |
```

Comment:
I like capturing this information per component, I do find reading this a bit difficult in its current format... I tried grouping components by signal and it made it a bit easier to read for me, let me know what you think. 
<img width="480" alt="Screen Shot 2021-09-15 at 2 50 23 PM" src="https://user-images.githubusercontent.com/223565/133514723-c37bdf87-aea1-4ba4-94e6-ea6e9ba8ace3.png">

If you think it's easier to read, here's the markdown for it:
```
| Signal | Component | Status |
|--------|-----------|--------|
|Trace   | OTLP protocol | Stable |
|| OTLP receiver functionality | Stable |
|| OTLP receiver configuration | Beta |
|| OTLP exporter functionality | Stable |
|| OTLP exporter configuration | Beta |
|Metric  | OTLP protocol | Stable |
|| OTLP receiver functionality | Stable |
|| OTLP receiver configuration | Beta |
|| OTLP exporter functionality | Stable |
|| OTLP exporter configuration | Beta |
|Logs    | OTLP protocol | Beta |
|| OTLP receiver functionality | Beta |
|| OTLP receiver configuration | Beta |
|| OTLP exporter functionality | Beta |
|| OTLP exporter configuration | Beta |
|Common| Logging exporter | Experimental |
|| Batch processor functionality | Beta |
|| Batch processor configuration | Beta |
|| MemoryLimiter processor functionality | Beta |
|| MemoryLimiter processor configuration | Beta |
```